### ISSUE TITLE
Link description for an image link is not proper

### ISSUE THREAD

**Issue by jose-cds** [cds-snc/platform-forms-client#3089]
**HC Assessment Label: Serious**

Pages:  Advancing Accessibility Standards Research Program Expression of Interest, Thank you
URL: https://forms-staging.cdssandbox.xyz/en/id/134
https://forms-staging.cdssandbox.xyz/id/134/confirmation
User agent:  Chrome (v119.0.6045.200)
Language: English and French

Issue Description: 
The text alternative or the value of the "aria-label" attribute provided for provided for the "Government of Canada" image link is not correct. For example, see the code snippet below.

<a href="http://canada.ca/en.html" aria-label="Symbol of the Government of Canada"><picture><img src="/img/sig-blk-en.svg" alt="Symbol of the Government of Canada" class="max-h-[40px] "></picture></a> 

<a href="http://canada.ca/fr.html" aria-label="Symbole du gouvernement du Canada"><picture><img src="/img/sig-blk-fr.svg" alt="Symbole du gouvernement du Canada" class="max-h-[40px] "></picture></a> 

Steps to reproduce:

1. Open the above URL in the Chrome browser
2. Inspect the HTML source for the image link

Expected result: The image link should have the appropriate text alternative to identify the purpose of the image link to users. In addition, the "aria-label" attribute in the "<a>" is redundant and should be removed.
For example, refer to this URL: https://design.canada.ca/common-design-patterns/signature.html

**Comment by srtalbot** [comment]
Fixed: <img width="1126" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOXkxQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--16e0c319cebe050d2b84f21b8e9141e8fbdc851e/image.png" alt="image.png" />
